src/test/java/bio/terra/stairway/QueueTest.java

```diff
@@ -0,0 +1,76 @@
+package bio.terra.stairway;
+
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+@Tag("connected")
+public class QueueTest {
+    private Logger logger = LoggerFactory.getLogger("bio.terra.stairway");
```

Comment:
nit: move away from string-based

Reply:
I searched for all of them and fixed them all.